src/components/middle/immedia/Immedia.scss

```diff
@@ -0,0 +1,68 @@
+.MainImmedia {
+  width: 100%;
+}
+
+.ImmediaHeader {
+  min-height: 110px;
+  max-height: 180px;
+  background: transparent;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  width: 100%;
+}
+
+.ImmediaBackground {
+  display: none;
+}
+
+.Participants {
+  clear: both;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.videoStream {
+  display: none;
+}
+
+.VideoName {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 2px;
```

Comment:
These small pixel values sound a little strange. I think it would be a better use of space if they are not present, and the canvas video and nickname are right next to eachother (or on top of eachother, see below), without additional margins.

Tweaked on dev tools:
![Screenshot from 2021-12-22 17-17-52](https://user-images.githubusercontent.com/14120807/147150530-31f78730-dce8-473f-9330-d1dd863b1293.png)

For reference, currently we have the nicknames on top of the cameras, with a semi transparent black background.
![Screenshot from 2021-12-22 17-19-38](https://user-images.githubusercontent.com/14120807/147150693-0473a30d-ff09-4d98-a973-b41f6c32693d.png)

What do you think?

Reply:
I was thinking about implementing some nickname overlay over the pictures but after testing I think it's better the current implementation. I removed all the margins and only kept a small margin to separate different participant rows:

I.e, with margin

![Screenshot from 2021-12-23 10-42-32](https://user-images.githubusercontent.com/11933424/147249658-d40a502f-c332-40db-b943-0d96df61d0bd.png)

and without it

![Screenshot from 2021-12-23 10-40-42](https://user-images.githubusercontent.com/11933424/147249704-7b14a970-25ca-406c-bf07-b5a575ca7db8.png)

This is some code I tested but, although it occupied less space, it was not as nice aesthetically

![Screenshot from 2021-12-23 10-22-41](https://user-images.githubusercontent.com/11933424/147249890-584131ff-1d4f-483d-a431-22f9616db17c.png)

Reply:
Sounds great that way!